src/openms/source/ANALYSIS/MAPMATCHING/ConsensusMapNormalizerAlgorithmMedian.cpp

```diff
@@ -79,37 +79,54 @@ namespace OpenMS
         feature_int[f_it->getMapIndex()].push_back(f_it->getIntensity());
```

Comment:
is it guaranteed that Map indices are always consecutive and start at 0?
otherwise this would segfault on certain datasets...